common/src/main/java/com/microsoft/identity/common/internal/request/AdalBrokerRequestAdapter.java

```diff
@@ -165,7 +165,7 @@ public BrokerInteractiveTokenCommandParameters brokerInteractiveParametersFromAc
                 .extraQueryStringParameters(extraQP)
                 .authority(authority)
                 .scopes(scopes)
-                .clientId(AuthenticationConstants.Broker.ACCOUNT_CLIENTID_KEY)
+                .clientId(intent.getStringExtra(AuthenticationConstants.Broker.ACCOUNT_CLIENTID_KEY))
```

Comment:
How are the requests working without a client id or how did they work :roll_eyes:  ?
Can you add a desc of the issue?

Reply:
Oh okay this is a regression due to `lombok` related changes, so looks like we haven't tested ADAL with broker for these changes.

Reply:
Yeah we haven't tested ADAL with Broker since my Lombok changes. I believe we don't have a version of ADAL with the latest common so it should be working. I guess, Teams will run into issues as they dual stack

Reply:
@kreedula  added.

Reply:
@kreedula On top of what @shahzaibj said, prod ADAL uses 2.0.5, so they won't run into this issue.

Reply:
@rpdome I believe Teams can still run into the issue, right? I believe they use the `common` pulled by `MSAL` and they could run into this with the latest MSAL I believe. Have they reported anything?

Reply:
Oh, by 'they' I meant prod ADAL. Yes, Teams would still run into the issue (which is why I said "on top of what you said". Grammar mistake :D

Reply:
It's actually super easy to repro. My guess is that they haven't tried ADAL flow :D